[wg/social] Participation: Move section from scope, Remove Chair Election, Add SWICG for Invited Experts

plehegar opened the pull request:
This addresses the folllowing comments:
- "Workflow for Active Items" would probably be a better fit under "Participation" than under "Scope"
that same subsection refers to "co-chair" elections, which I don't think are in line with the Process
- In the participation section, and given the strong influence of the SWICG, it seems odd that the CG is not mentioned in the paragraph about potential Invited Experts. The charter template suggests adding that, if there is a relevant CG.

cc https://github.com/w3c/strategy/issues/435

plehegar committed:
Moved "Workflow for Active Items" in the participation section

plehegar committed:
Chairs are appointed by the Team

plehegar committed:
Point to the CG for potential IE experts

dmitrizagidulin commented:
@plehegar why the change to the Chairs policy? That wasn't requested by any reviewer in https://github.com/w3c/strategy/issues/435, if I understand correctly, was it?

plehegar commented:
> @plehegar why the change to the Chairs policy? That wasn't requested by any reviewer in [w3c/strategy#435](https://github.com/w3c/strategy/issues/435), if I understand correctly, was it?


The comment was:
[[
"Workflow for Active Items" would probably be a better fit under "Participation" than under "Scope"
that same subsection refers to "co-chair" elections, which I don't think are in line with the Process
]]


and the Process says:
[[
Each group must have a Chair (or co-Chairs) to facilitate effective discussion and coordinate the group’s activities. The [Team](https://www.w3.org/policies/process/20250818/#team) appoints (and re-appoints) Chairs for all groups.
]]
https://www.w3.org/policies/process/20250818/#ReqsAllGroups

dmitrizagidulin commented:
@plehegar understood, thanks!

dariusk committed:
Updating URL (#727)

This was pointing to an application link for IE for a different WG.
Changing so it points to the generic IE application instructions.
